plugin-packs/postcss-preset-env/src/lib/get-options-for-browsers-by-feature.mjs

```diff
@@ -45,6 +45,17 @@ export default function getOptionsForBrowsersByFeature(browsers, feature, cssdbL
 				}
 			}
 
+			return {};
+		case 'logical-properties-and-values':
+		case 'float-clear-logical-values':
+		case 'logical-resize':
+		case 'logical-viewport-units':
+			{
+				if ('logical' in options) {
+					return options.logical;
+				}
+			}
```

Comment:
After a lot of thinking about where this should live, I thought this was the best place but let me know if you disagree!

Reply:
At first I disagreed because I thought we had a thing for shared plugin options.
But these options are not intended for all plugins, only a select few. This is different from `preserve`.

So I think this is indeed the best way to do this and the best place for this code 🎉